modules/azure_functions/src/azure_functions.c

```diff
@@ -393,7 +393,8 @@ static void AzureFunctions_Receive(MODULE_HANDLE moduleHandle, MESSAGE_HANDLE me
                                             {
 
                                                 //Add here the content on the body.
-                                                BUFFER_HANDLE postContent = BUFFER_create(STRING_c_str(jsonToBeAppended), STRING_length(jsonToBeAppended));
+                                                size_t appendLength = STRING_length(jsonToBeAppended);
```

Comment:
The important thing was that we couldn't have more than one function call in the arguments to `BUFFER_create` because of platform differences RE the order in which arguments are popped off the stack. Unit tests that use extensive mocking + EXPECTED_CALL will pass on one platform and fail on another. ☹️